CONTRIBUTING.md

```diff
@@ -53,9 +53,13 @@ In this repo we use [pre-commit](https://pre-commit.com/) hooks for automated va
 
 > ℹ️ **HINT:** *Consider [automatically enabling the hooks in every Git repository](https://pre-commit.com/#automatically-enabling-pre-commit-on-repositories)*
 
-### End2End Testing
+### Code Testing
 
-Our E2E tests can be found in the `/test` folder and follow the journey of someone as they would use the Zarf CLI.  In CI these tests run against our currently supported cluster distros.  You can learn more about testing of Zarf [here](https://docs.zarf.dev/docs/developer-guide/testing).
+Our E2E tests can be found in the `/test` folder and follow the journey of someone as they would use the Zarf CLI.  In CI these tests run against our currently supported cluster distros, and are the primary way that Zarf code is tested.
+
+Our Unit tests can be found as `*_test.go` files inside the package that they are designed to test.  These are also run in CI and are designed to test small functions with clear interfaces that would be difficult to test otherwise.
```

Comment:
Probably should mention we only have unit tests in `src/pkg`, especially since the make target reflects that as well.

Reply:
done